crypto/x509/v3_crld.c

```diff
@@ -264,9 +264,10 @@ static void *v2i_crld(const X509V3_EXT_METHOD *method,
                 goto err;
             point = crldp_from_section(ctx, dpsect);
             X509V3_section_free(ctx, dpsect);
-            if (point == NULL)
+            if (point == NULL
+                /* no failure as it was reserved */
+                || !ossl_assert(sk_DIST_POINT_push(crld, point)))
```

Comment:
Same as elsewhere I would not change this. We can depend on no failure on pre-reserved pushes.